### DISCUSSION TITLE
No option to create payment on Zettle - URGENT!

### DISCUSSION THREAD

**Discussion by stickybrickit** [Zettlr/Zettlr#2440] [Q&A]
I bought the iZettle a couple of weeks ago and set the app up on my iPhone 7. As a back-up when we do events I also got my partner to install it on her phone (Samsung Galaxy). When I open the app on my phone I have as the second option in the list as "Amount" which is how I charge money via the device. However on my partners Samsung there is no "Amount" option so we cannot use her phone with the reader.

Both phones are paired with the device and I can log into my account fine on both. I have tried uninstalling / reinstalling the app on both devices and the Amount option never appears on her phone. I have a private account and understand I can't use several devices at the same time, but that I ought to be able to use one at once (hence logging out / uninstalling the app etc, but this never seems to have worked).

Anyone had this before and any ideas how to fix it? We have an event in a couple of days so need to get this resolved ASAP!!!

**Comment by nathanlesage** [comment]
As you already heard on Reddit, iZettle ≠ Zettlr. Please refer to their homepage and/or issue tracker instead.

Locking the discussion.
